com.gitlab.ColinDuquesnoy.MellowPlayer.yml

```diff
@@ -0,0 +1,37 @@
+app-id: com.gitlab.ColinDuquesnoy.MellowPlayer
+runtime: org.kde.Platform
+runtime-version:  "5.15"
+base: io.qt.qtwebengine.BaseApp
+base-version: "5.15"
+sdk: org.kde.Sdk
+command: MellowPlayer
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --device=dri
+  - --socket=session-bus
+  - --socket=system-bus
+  - --filesystem=/tmp
```

Comment:
Does it need to share `/tmp` with the host?

Reply:
I'll check that

Reply:
I've removed --filesystem=/tmp